src/renderer/mod.rs

```diff
@@ -83,9 +83,51 @@ impl<'a> Renderer<'a> {
 
         #[inline]
         fn find_variable(context: &Value, key: &str, tpl_name: &str) -> Result<Value> {
-            match context.pointer(&get_json_pointer(key)) {
+            let sub_vars_to_calc = pull_out_square_bracket(key);
+            let mut new_key = key.to_string();
+
+            for sub_var in sub_vars_to_calc.iter() {
+                // Translate from variable name to variable value
+                let post_var = find_variable(context, sub_var.as_ref(), tpl_name).unwrap();
+                let post_var_as_str = {
+                    if post_var.is_string() {
+                        post_var.as_str().unwrap().to_string()
+                    } else if post_var.is_boolean() {
+                        post_var.as_bool().unwrap().to_string()
+                    } else if post_var.is_i64() {
```

Comment:
add `f64` as well maybe? since that's the result of math operations in Tera

Reply:
Yep. Fixing.